README.md

```diff
@@ -40,16 +40,10 @@ For example, a document definition file implemented as an object:
       myDocType1: {
         channels: {
           view: 'view',
-          add: 'add',
-          replace: 'change',
-          remove: 'delete'
+          write: 'write'
         },
-        typeFilter: function(doc, oldDoc) {
-          if (oldDoc) {
-            return doc.type === oldDoc.type && oldDoc.type === 'myDocType1';
-          } else {
-            return doc.type === 'myDocType1';
-          }
+        typeFilter: function(doc, oldDoc, docType) {
+          return oldDoc ? oldDoc.type === docType : doc.type === docType;
```

Comment:
While I was in here, I figured I'd simplify/shorten these examples somewhat to decrease the mental burden on a first-time reader of the doc.